contributing.md

```diff
@@ -76,6 +76,8 @@ languages in Bazel right now.
       cd bazel
       bazel --output_user_root=c:\tmp build //src:bazel.exe
       ```
+      
+      For faster iteration times (but larger binaries), use `//src:bazel-dev.exe` instead.
```

Comment:
Since it seems better overall, we can use this target in the example above (people like to copy-paste instructions).

Reply:
It's not the binary that we ship, that's why I was hesitating. If you don't think it matters, I'll change it.

Reply:
As you prefer